pkgs/development/compilers/gerbil/default.nix

```diff
@@ -1,12 +1,12 @@
 { stdenv, callPackage, fetchurl, gambit }:
 
 callPackage ./build.nix {
-  version = "0.13";
-  git-version = "0.13";
+  version = "0.14";
+  git-version = "0.14";
   GAMBIT = gambit;
   SRC = fetchurl {
```

Comment:
Why not use `fetchFromGitHub`?

Reply:
done.

Reply:
this is still fetchurl

Reply:
Oh, I had done it for unstable, but not default. Ok, done for default (in my next update, as soon as it builds).